cfgov/unprocessed/css/organisms/info-unit-group.less

```diff
@@ -46,15 +46,10 @@
 .o-info-unit-group {
     // Set correct spacing for consecutive headings in the intro per DS specs:
     // https://cfpb.github.io/design-system/foundation/headings#behavior
-    > h2 + .rich-text > h3:first-child {
+    > h2 > h3:first-child {
```

Comment:
I think this rule can go away entirely, like the ones below, because they were only necessary due to the presence of the `.rich-text` interloper.